src/Model/Domain/TaskerPool/TaskerPoolRepository.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace G4\Tasker\Model\Domain\TaskerPool;
+
+class TaskerPoolRepository
+{
+    const TABLE_NAME = 'tasker_pool';
+
+    /**
+     * @var \PDO
+     */
+    private $pdo;
+
+    public function __construct(\PDO $pdo)
+    {
+        $this->pdo = $pdo;
+    }
+
+    public function upsert(string $taskerPoolEntity)
+    {
+        $query = sprintf(
+            "INSERT INTO " . self::TABLE_NAME . "(`hostname`,`status`,`ts_available`) VALUES ('%s',%s,%s) ON DUPLICATE KEY UPDATE ts_available=UNIX_TIMESTAMP();",
```

Comment:
Improve readability

Change this
`"INSERT INTO " . self::TABLE_NAME . "(`hostname`,`status`,`ts_available`) VALUES ('%s',%s,%s) ON DUPLICATE KEY UPDATE ts_available=UNIX_TIMESTAMP();",`

to 
`"INSERT INTO %s (`hostname`,`status`,`ts_available`) VALUES ('%s',%s,%s) ON DUPLICATE KEY UPDATE ts_available=UNIX_TIMESTAMP();",`